skimage/filters/rank/tests/test_rank.py

```diff
@@ -41,8 +41,7 @@ def test_otsu_edge_case():
     assert result[1, 1] in [141, 172]
 
 
-
```

Comment:
@clementkng do you have an editor that lets you pick out the lines of code you commit? I know you are "cleaning" this function, but this is something I consider a "flavour of the week" and has a chance to detract from your other contribution.

Reply:
I adopted this PR from @Henley13, who originally made these changes. I can try reversing them.

Reply:
no not worth it.